internal/oapispec/ffi2swagger.go

```diff
@@ -35,9 +35,35 @@ func NewFFISwaggerGen() FFISwaggerGen {
 	return &ffiSwaggerGen{}
 }
 
-func (og *ffiSwaggerGen) Generate(ctx context.Context, baseURL string, ffi *fftypes.FFI) (*openapi3.T, error) {
+func (og *ffiSwaggerGen) Generate(ctx context.Context, baseURL string, ffi *fftypes.FFI) (swagger *openapi3.T, err error) {
 
 	// routes := []*Route{}
 
+	// for _, method := range ffi.Methods {
+	// 	if routes, err = og.addMethod(ctx, routes, method); err != nil {
+	// 		return nil, err
+	// 	}
+	// }
+
+	// for _, event := range ffi.Events {
+	// 	if routes, err = og.addEvent(ctx, routes, event); err != nil {
+	// 		return nil, err
+	// 	}
+	// }
+
 	return nil, nil
 }
+
+// func (og *ffiSwaggerGen) addMethod(ctx context.Context, routes []*Route, method *fftypes.FFIMethod) ([]*Route, error) {
+
+// 	routes = append(routes, &Route{
+// 		Name: fmt.Sprintf("method_%s_invoke", method.Name),
+// 	})
+
+// 	return routes, nil
+// }
+
+// func (og *ffiSwaggerGen) addEvent(ctx context.Context, routes []*Route, event *fftypes.FFIEvent) ([]*Route, error) {
+
+// 	return routes, nil
+// }
```

Comment:
Is this going to be uncommented at some point, or should it be removed?

Reply:
Either way - this was the key thing I wanted to work on (was hoping to have it done by today sorry), but needed the unique pathnames, which took me down the rabbit hole with the other changes.

Reply:
I submitted the PR early into the working branch, as I felt I needed to discuss with @awrichar the approach on the events.

Reply:
Ah got it. We can leave it for now, since this is just a PR into the feature branch